docs/self-hosted-appcircle/install-server/helm-chart/installation/docker-k8s-migration.md

```diff
@@ -0,0 +1,1343 @@
+
+### 3. Prepare `values.yaml` for Migration
+
+**Update your `values.yaml` file** with the following configurations before deploying the Helm chart to ensure a smooth migration process.
+
+---
+
+#### **Storage Updates**
+
+After [checking the data size](#3-check-the-data-size-on-the-standalone-appcircle-server) on the standalone Appcircle server, configure the storage sizes in `values.yaml` to meet or exceed your current data size requirements. Adequate storage is essential for a successful migration and seamless operation of the Appcircle server.
+
+For detailed instructions, refer to the [storage configuration](/self-hosted-appcircle/install-server/helm-chart/configuration/storage-configuration) page.
+
+---
+
+#### **Keycloak Updates**
+
+Ensure the `organizationName` and `initialOrganizationId` in the `values.yaml` file match those in the standalone Appcircle server.
+
+To locate these values:
+1. Go to the standalone Appcircle server dashboard.
+2. Switch to the root organization.
+3. Open the developer tools and switch to the "Network" tab.
+4. Navigate to your **Organization** page to retrieve the details.
+5. You can directly see the organization name in the UI.
+6. For the organization ID, filter the `userinfo` request.
+7. In the response, check for `currentOrganizationId`.
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE-4705-organization-page.png' />
+
+Here’s an example configuration for `values.yaml`:
+```yaml
+auth:
+  auth-keycloak:
+    replicas: 0
+    organizationName: spacetech
+    initialOrganizationId: fb**************a7
+```
+
+:::note
+`auth-keycloak.replicas: 0`** disables Keycloak during the migration to avoid authentication conflicts.
+:::
+---
+
+#### **MongoDB Updates**
+
+Set the resources preset to `"large"` for the migration phase to provide MongoDB with sufficient resources for handling larger datasets.
+
+Example configuration:
+```yaml
+mongodb:
+  resourcesPreset: "large"
+```
+
+---
+
+#### **KVS Subdomain**
+
+The `redis` subdomain used on the standalone Appcircle server has been updated to `kvs` in the Helm configuration for the Kubernetes deployment.
+
+##### If you want to update to the `kvs` subdomain:
+
+- **No changes are needed in the `values.yaml`** file.
+- Add the **`kvs` DNS record** for the new subdomain.
+- Update the **Appcircle runner configurations** to point to the new `kvs` subdomain.
+
+##### If you prefer to keep the `redis` subdomain (recommended):
+
+- Add the following configuration to the `values.yaml` file:
+  
+  ```yaml
+  global:
+    urls:
+      webEventRedis:
+        subdomain: redis
+  ```
+
+- **No new DNS record is required**.
+- **Appcircle runners do not need to be updated**.
+
+#### **Additional Updates**
```

Comment:
Before additional updates section, we should mention about SSL configuration.

Reply:
SSL configuration page should have notice that warns user about the last might not be rootCA in all cases. For example Lets Encrypt case.